two-node-cluster/setup/setup-nginx.sh

```diff
@@ -74,4 +74,9 @@ echo "============================================"
 sudo sed -i 's$server xxx.xxx.xxx.1:9443$server '$wso2_is_1_ip':9443$g' /etc/nginx/conf.d/is.conf || echo "error 1"
 sudo sed -i 's$server xxx.xxx.xxx.2:9443$server '$wso2_is_2_ip':9443$g' /etc/nginx/conf.d/is.conf || echo "error 1"
 
+echo ""
+echo "Adding workerconnection to nginx.conf file"
+echo "============================================"
+sudo sed -i 's/worker_connections 768/worker_connections 1500/g' /etc/nginx/nginx.conf || echo "error 1"
```

Comment:
Why do we need this?

Reply:
nginx throw 'worker_connections are not enough' alert for two node and 4 core perfroamnce test execution with 500 concurrency . Default worker connection number is 512 and currently using the default one.